foundry/foundry.py

```diff
@@ -380,18 +381,19 @@ def download(self, globus=True, verbose=False, **kwargs):
             crawl_url = f"{xtract_base_url}/crawl"
             if verbose:
                 print(f"Crawl URL is : {crawl_url}")
+
+            first_ep_dict = {
```

Comment:
what does "first ep" refer to? I see in a comment above it's some kind of crawling service? oh so is this the first set of search criteria for the crawler (with the idea that there will be other queries in the future)?

Reply:
This is terminology from Tyler, I am not sure why "first" is descriptive here as there is no "second" anywhere. I'll change this to `ep_dict`

Reply:
thanks Ben!